Package.swift

```diff
@@ -10,20 +10,39 @@ let package = Package(
         // Products define the executables and libraries a package produces, and make them visible to other packages.
         .library(
             name: "SecureStore",
-            targets: ["SecureStore"])
+            targets: ["SecureStore"]
+        ),
+        .library(
+            name: "CryptoService",
+            targets: ["CryptoService"]
+        )
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/attaswift/BigInt", from: "5.3.0")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .target(
-            name: "SecureStore"
+            name: "SecureStore",
+            dependencies: ["CryptoService"]
```

Comment:
open question: is this dependency needed? It feels like it might not be (at least not yet?)
I wonder if we can / should keep it loosely coupled with dependency inversion (dependency based on abstraction - a protocol)

Reply:
It isn't needed yet, no.
I've removed that for the time being.
Would we not still need to add this dependency when we need to make use of the `CryptoEncryptionService` in `SecureStoreService`?